src/hardware/serialport/softmodem.h

```diff
@@ -61,105 +61,100 @@ bool MODEM_ReadPhonebook(const std::string &filename);
 
 class CFifo {
 public:
-	CFifo(Bitu _size)
-		: data(_size),
-		  size(_size),
-		  pos(0),
-		  used(0)
-	{}
-
-	INLINE Bitu left(void) const {
-		return size - used;
-	}
-	INLINE Bitu inuse(void) const {
-		return used;
-	}
-	void clear(void) {
+	CFifo(uint32_t _size) : data(_size), size(_size), pos(0), used(0) {}
```

Comment:
Except for the `uint8_t` port index (down from a `Bitu`), this file only contains 1:1 type replacements and member value assignment.